android/tests_backend/widgets/properties.py

```diff
@@ -22,7 +22,7 @@ def toga_color(color_int):
         )
 
 
-def toga_alignment(gravity, justification_mode=None):
+def to_toga_text_align(gravity, justification_mode=None):
```

Comment:
While changing all instances of `text_alignment` to `text_align`, I've also added `to_` to the beginning of converters like this, to help avoid the ambiguity of "align" being a verb.

Reply:
FWIW - I'm not a big fan of "verbed" function names (e.g., `to_` or `convert_` prefixes) when it comes to simple format conversion methods. My take is that if the variable by itself would be called `foobar`, then then method should be called `foobar(original)`. However this is a "strong opinion, weakly held", and it isn't public API, so I'm not *overly* concerned.

Reply:
Is there another form you'd prefer, that makes it similarly clear the function itself isn't aligning text? If it weren't for that, that's the model I would've followed too.

Reply:
Honestly - `toga_text_align()` doesn't bother me as a name. Yes, it's "verb ambiguous"... but I'd argue that if you were going to "do the alignment", the method should be named `align_text()`.

Reply:
...That is a fair point. Perhaps I've just been overthinking this. I found it confusing myself, but to be fair, search-and-replace across over a hundred files is neither the clearest nor the most common way to read source code; I'll remove the extraneous `to_`s, and I'm sure no noses will lose any skin over it.